x-pack/plugins/cloud_security_posture/public/pages/findings/layout/findings_layout.tsx

```diff
@@ -65,86 +72,164 @@ export const getExpandColumn = <T extends unknown>({
   ],
 });
 
-export const getFindingsColumns = (): Array<EuiBasicTableColumn<CspFinding>> => [
-  {
-    field: 'resource.id',
-    name: (
-      <ColumnNameWithTooltip
-        columnName={TEXT.RESOURCE_ID}
-        tooltipContent={i18n.translate(
-          'xpack.csp.findings.findingsTable.findingsTableColumn.resourceIdColumnTooltipLabel',
-          {
-            defaultMessage: 'Custom Elastic Resource ID',
-          }
-        )}
+export const getFindingsColumns = ({
+  onAddFilter,
+}: {
+  onAddFilter?(key: string, value: Serializable, negate: boolean): void;
+} = {}): Array<EuiTableFieldDataColumnType<CspFinding>> =>
+  [
+    {
+      field: 'resource.id',
+      name: (
+        <ColumnNameWithTooltip
+          columnName={TEXT.RESOURCE_ID}
+          tooltipContent={i18n.translate(
+            'xpack.csp.findings.findingsTable.findingsTableColumn.resourceIdColumnTooltipLabel',
+            {
+              defaultMessage: 'Custom Elastic Resource ID',
+            }
+          )}
+        />
+      ),
+      truncateText: true,
+      width: '15%',
+      sortable: true,
+      render: (filename: string) => (
+        <EuiToolTip position="top" content={filename}>
+          <span>{filename}</span>
+        </EuiToolTip>
+      ),
+    },
+    {
+      field: 'result.evaluation',
+      name: TEXT.RESULT,
+      width: '120px',
+      sortable: true,
+      render: (type: PropsOf<typeof CspEvaluationBadge>['type']) => (
+        <CspEvaluationBadge type={type} />
+      ),
+    },
+    {
+      field: 'resource.sub_type',
+      name: TEXT.RESOURCE_TYPE,
+      sortable: true,
+      width: '150px',
+    },
+    {
+      field: 'resource.name',
+      name: TEXT.RESOURCE_NAME,
+      sortable: true,
+    },
+    {
+      field: 'rule.name',
+      name: TEXT.RULE,
+      sortable: true,
+    },
+    {
+      field: 'rule.section',
+      name: TEXT.CIS_SECTION,
+      sortable: true,
+      truncateText: true,
+    },
+    {
+      field: 'cluster_id',
+      name: (
+        <ColumnNameWithTooltip
+          columnName={TEXT.CLUSTER_ID}
+          tooltipContent={i18n.translate(
+            'xpack.csp.findings.resourceTable.resourceTableColumn.clusterIdColumnTooltipLabel',
+            {
+              defaultMessage: 'Kube-System Namespace ID',
+            }
+          )}
+        />
+      ),
+      truncateText: true,
+      sortable: true,
+    },
+    {
+      field: '@timestamp',
+      width: '150px',
+      name: TEXT.LAST_CHECKED,
+      truncateText: true,
+      sortable: true,
+      render: (timestamp: number) => (
+        <EuiToolTip position="top" content={timestamp}>
+          <span>{moment(timestamp).fromNow()}</span>
+        </EuiToolTip>
+      ),
+    },
+  ].map((column) => (onAddFilter ? createColumnWithFilters(column, { onAddFilter }) : column));
```

Comment:
we know all the columns for this table (`latest_findings` and `findings_resource`) are filterable, so we `map` on all of them to  `createColumnWithFilters.